mindsdb/integrations/handlers/bigquery_handler/Manual_QA.md

```diff
@@ -5,8 +5,10 @@
 ## Testing Bigquery Handler with [Dataset Name](URL to the Dataset)
 
 **1. Testing CREATE DATABASE**
-
+![Capture1](https://github.com/mindsdb/mindsdb/assets/70659811/68352266-1de5-4a31-81bc-f206f26fe84f)
```

Comment:
This error is expected. The error message clearly states that you need to provide either `service_account_keys` or `service_account_json`. For details, see here: https://docs.mindsdb.com/data-integrations/google-bigquery